jf-common/jf-common-vendor-blobs.mk

```diff
@@ -152,4 +152,6 @@ PRODUCT_COPY_FILES += \
 	vendor/samsung/jf-common/proprietary/vendor/lib/libwvdrm_L1.so:system/vendor/lib/libwvdrm_L1.so \
 	vendor/samsung/jf-common/proprietary/vendor/lib/libwvm.so:system/vendor/lib/libwvm.so \
 	vendor/samsung/jf-common/proprietary/vendor/lib/libWVStreamControlAPI_L1.so:system/vendor/lib/libWVStreamControlAPI_L1.so \
-	vendor/samsung/jf-common/proprietary/vendor/lib/mediadrm/libwvdrmengine.so:system/vendor/lib/mediadrm/libwvdrmengine.so
+	vendor/samsung/jf-common/proprietary/vendor/lib/mediadrm/libwvdrmengine.so:system/vendor/lib/mediadrm/libwvdrmengine.so \
+	vendor/samsung/jf-common/proprietary/lib/libstlport.so:system/lib/libstlport.so \
+	vendor/samsung/jf-common/proprietary/lib/libgnustl_shared:system/lib/libgnustl_shared
```

Comment:
I can try it without it and see if the sensors still work. Not sure what its supposed to do, but something will break if its needed.